employee-onboarding-A2A-reference-implementation/src/apis/hr_api.py

```
@@ -0,0 +1,323 @@
+        manager_email=str(employee.manager_email),
+        start_date=employee.start_date.isoformat(),
+        status="pending_onboarding",
+        created_at=now,
+        created_by=token.sub,
+    )
+
+
+@router.get("/employees/{employee_id}", response_model=EmployeeResponse)
+async def get_employee(
+    employee_id: str,
+    token: TokenClaims = Depends(validate_token)
```

Ruff (0.15.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)